CHANGES.rst

```diff
@@ -13,6 +13,22 @@ Changes
   The meta data us useful, for example, for selecting tests with the
   nose attribute mechanism.
 
+- Added ``doctestcase.doctestfiles``
+
+  - Define multiple doctest files at once.
+
+  - Automatically assign test class members.  So rather than::
+
+      class MYTests(unittest.TestCase):
+          ...
+          test_foo = doctestcase.doctestfile('foo.txt')
+
+    You can use::
+
+      @doctestcase.doctestfiles('foo.txt')
+      class MYTests(unittest.TestCase):
+          ...
+
```

Comment:
I suggest that the utility of this feature would be more easily apparent if the examples showed at least two doctest files, i.e.

```python
    class MyTests(unittest.TestCase):
        ...
        test_foo = doctestcase.doctestfile('foo.txt')
        test_bar = doctestcase.doctestfile('bar.txt')
        ...
```
and
```python
    @doctestcase.doctestfiles('foo.txt', 'bar.txt', ...)
    class MyTests(unittest.TestCase):
        ...
```